Dockerfile

```diff
@@ -0,0 +1,49 @@
+# This file is used for building and testing agent.
```

Comment:
Could you add the standard license header?

Reply:
OK